src/screens/EventDetailsScreen/IconItem.js

```diff
@@ -2,28 +2,40 @@
 import React from "react";
 import { View, StyleSheet } from "react-native";
 import type { Node } from "react";
+import type { StyleObj } from "react-native/Libraries/StyleSheet/StyleSheetTypes";
 import Text from "../../components/Text";
 import type { TextType } from "../../components/Text";
+import { lightNavyBlueColor } from "../../constants/colors";
 
 type IconItemProps = {|
   icon: Node,
   title: string,
   content?: Node,
-  titleType?: TextType
+  titleType?: TextType,
+  textStyle?: StyleObj
 |};
 
-const IconItem = ({ icon, title, titleType, content }: IconItemProps) => (
+const IconItem = ({
+  icon,
+  title,
+  titleType,
+  content,
+  textStyle
+}: IconItemProps) => (
   <View style={styles.iconItem}>
     <View style={styles.icon}>{icon}</View>
     <View style={styles.item}>
-      <Text type={titleType}>{title}</Text>
+      <Text type={titleType} style={[styles.title, textStyle]}>
```

Comment:
I suggest removing this from here and just accepting children. The location will be a link so it makes it much more flexible if these are passed in.

Reply:
For example this would become:
```
<View style={styles.item}>{children}</View>
```

Reply:
This simplifies the props required here and opens this up to being reused in more contexts.

Reply:
@RGBboy We've refactored, please can you take another look.